CHANGELOG.md

```diff
@@ -3,8 +3,10 @@
 ### Breaking Changes
 
 ### Deprecations
+- `--tx-pool-disable-locals` has been deprecated for removal in favor of `--tx-pool-no-local-priority`, no semantic change, only a renaming [#5959](https://github.com/hyperledger/besu/pull/5959)
```

Comment:
Is this from another PR? It doesn't seem related to the price-bump behaviour.

Reply:
removed, this is not related to this PR